assets/js/modules/adsense/components/settings/SettingsForm.js

```diff
@@ -166,12 +143,6 @@ export default function SettingsForm() {
 
 			<AutoAdExclusionSwitches />
 
-			{ existingAdBlockingRecoveryTag && (
-				<SettingsNotice
-					notice={ existingAdBlockingRecoveryTagMessage }
-				/>
-			) }
-
```

Comment:
Moved to the right place consistent with the notice for AdSense existing tag. This was also being rendered unconditionally instead of being guarded by the feature flag.